packages/office-addin-manifest/src/manifestHandler/manifestHandlerJson.ts

```diff
@@ -1,29 +1,61 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
+import { TeamsAppManifest } from "@microsoft/teamsfx-api";
+import { ManifestUtil } from "@microsoft/teams-manifest";
+import { v4 as uuidv4 } from "uuid";
 import { ManifestInfo } from "../manifestInfo";
 import { ManifestHandler } from "./manifestHandler";
 
 export class ManifestHandlerJson extends ManifestHandler {
   /* eslint-disable @typescript-eslint/no-unused-vars */
-  async modifyManifest(guid?: string, displayName?: string): Promise<any> {
-    throw new Error("Manifest cannot be modified in .json files");
+  async modifyManifest(guid?: string, displayName?: string): Promise<TeamsAppManifest> {
+    const teamsAppManifest: TeamsAppManifest = await JSON.parse(this.fileData);
```

Comment:
Isn't the new library supposed to have manifest reading as part of it?

Reply:
Sure, I just added the reading with the library now